### PR TITLE
Website to test our knowledge in basic html and css

### PR DESCRIPTION
**Pull request by MarkYegon7**
In this project I:

- Able to use basic HTML tags
- Styling using css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+
+    <link rel="stylesheet" href="index.css">
+    <title>YouTube Replica</title>
+</head>
+<body>
+
+    <header class="header">
```
Comment (MarkYegon7) [inline index.html:17]: I think :thinking: the use of class names similar to HTML tags can cause confusion, especially when working with CSS.Try using unique class names.:blush:

File: index.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                    Inside Google's Massive Headquarters <br> Tech Vision
+                </h3>
+            </div>
+
+            <!-- <div class="title">
+                <iframe width="560" height="315" src="https://www.youtube.com/embed/TcqAu8VyjZA" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+                <h3>
+                    Uzbekistan - The Silk Road by train | DW Documentary <br> DW Documentary
+                </h3>
+            </div> -->
```
Comment (MarkYegon7) [inline index.html:83]: Please do not leave unused code as a comment in your work. This makes your code unclean.:thumbsup:

**Review by MarkYegon7 — COMMENTED** [review]
# Changes required :recycle: 
giphy

Good job so far! I really appreciate the effort you've put in :100: Please check the comments below and make the required changes.
Thanks :blush:

**Review by Olamarx — APPROVED** [review]
Hi @MarkYegon7,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Polite Comments ✔️ 
- Reason for comments (WHY) ✔️ 
- Clear and semantically correct sentences ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**